.github/workflows/phpunit-tests.yml

```diff
@@ -0,0 +1,144 @@
+name: PHPUnit Tests
+
+# Since Unit Tests are required to pass for each PR,
+# we cannot disable them for documentation-only changes.
+on:
+    pull_request:
+    push:
+        branches:
+            - trunk
+            - 'release/**'
+            - 'wp/**'
+
+# Cancels all previous workflow runs for pull requests that have not completed.
+concurrency:
+    # The concurrency group contains the workflow name and the branch name for pull requests
+    # or the commit hash for any other events.
+    group: ${{ github.workflow }}-${{ github.event_name == 'pull_request' && github.head_ref || github.sha }}
+    cancel-in-progress: true
+
+jobs:
+    # Runs the PHPUnit tests for Gutenberg.
+    #
+    # Performs the following steps:
+    # - Sets environment variables.
+    # - Checks out the repository.
+    # - Sets up Node.js.
+    # - Sets up PHP.
+    # - Installs Composer dependencies.
+    # - Installs npm dependencies
+    # - Logs general debug information about the runner.
+    # - Logs Docker debug information (about the Docker installation within the runner).
+    # - Starts the wp-env Docker containers.
+    # - Logs the running Docker containers.
+    # - Logs debug information about what's installed within the wp-env Docker containers.
+    # - Install WordPress and Gutenberg within the Docker container.
+    # - Run the PHPUnit tests.
+    test-php:
+        name: ${{ matrix.php }}${{ matrix.multisite && ' multisite' || '' }} on ${{ matrix.os }}
+        runs-on: ${{ matrix.os }}
+        timeout-minutes: 20
+        if: ${{ github.repository == 'WordPress/gutenberg' || github.event_name == 'pull_request' }}
+        strategy:
+            fail-fast: false
+            matrix:
+                php:
+                    [
+                        '5.6',
+                        '7.0',
+                        '7.1',
+                        '7.2',
+                        '7.3',
+                        '7.4',
+                        '8.0',
+                        '8.1',
+                        '8.2',
+                    ]
```

Comment:
Either specify it like so:
```yaml
php: ['5.6', '7.0', '7.1', '7.2', '7.3', '7.4', '8.0', '8.1', '8.2']
```
or like so:
```yaml
php:
  - '5.6'
  - '7.0'
  - '7.1'
  - '7.2'
  - '7.3'
  - '7.4'
  - '8.0'
  - '8.1'
  - '8.2'  
```

This is Yaml, not PHP.

Reply:
I agree with @jrfnl on this, but…

> This is Yaml, not PHP.

I don't see the reason to add snark in this comment. The code as written parses as expected and is valid YAML. We don't have to claim that @anton-vlasenko was unaware of which language is being written or put anyone down when their style is different than your style or even when we find stylistic differences in a project and a code submission.

Reply:
Sorry if you interpreted my comment as snarky, definitely wasn't meant as such. It was just meant to convey that I understand where the inclination to write it like that comes from, nothing more.

Reply:
> I understand where the inclination to write it like that comes from

I'm not the Gutenberg maintainer, so this isn't coming from any authority, in the way that you have expressed multiple times having authority yourself, but what I share here is meant to be a reminder of a way that help avoid these kinds of misunderstandings: we believe we understand the intentions and inclinations behind someone else's behaviors more than we really do understand, and leaning on that assumption is the source of many needless problems.

when we start by assuming we understand someone else's motivations it just about guarantees friction and trouble. we're not giving those people a chance to share their motivations and we're often starting by assuming they are wrong (which also means we're not giving ourselves an opportunity to learn from something that seems wrong at our quick glance).

it's fine to notice a style issue like this and share [as you did in the next comment](https://github.com/WordPress/gutenberg/pull/46510#discussion_r1053847254) but it's a big leap to start assuming we know what's inside someone else's head and then to assert it publicly, and then tell them that they are wrong because of the way we know they were thinking.

I've far too often had to deal with big problems that arise because of a "fix" that was made due to something looking out of place but being there for a reason, and the fixer comes in and changes it because they assume it needs to be removed, assume it was a mistake the author made. obviously we come across oddities and quirks all the time, and in most cases those are good points to revise code that was incidentally awkward, but I believe we can build stronger relationships if we start by asking _why_ something is different than we expect rather than telling someone "I know why you did this and it's wrong," especially if those declarations do end up being wrong and the cause for real failures that do bring a lot of frustration.

you bring a lot of good value to a review like this; I want to acknowledge that and appreciate your time and attention there. just want to ask at the same time if we can bring positive assumptions to the discussion, like assuming that people are smarter than we give them credit for, or have reasons behind the choices they make.

Reply:
Please take your own advise.

Reply:
And just to be crystal clear what I mean: you seem to be making all sorts of assumptions about me and are interpreting my comments with that in your mind and looking for meaning behind the words, where there is none. My comments are plain and should be taken at face-value. Nothing more. I can't help it if someone reads something into them what isn't there.

Reply:
I don't see @jrfnl 's comment as snarky either. 
I sincerely appreciate your expertise and your code review, @jrfnl.
Thank you!